its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ServerTests.java

```diff
@@ -89,6 +96,23 @@ private static void addPlugin(OrchestratorExtensionBuilder orchestrator, String
     if (version == null || version.isEmpty() || version.equals("NONE")) {
       return;
     }
-    orchestrator.addPlugin(TestUtils.getMavenLocation(groupId, artifactId, version));
+    orchestrator.addPlugin(MavenLocation.of(groupId, artifactId, version));
+  }
+
+  private static Path customRoslynPlugin() {
```

Comment:
Moved from TestUtils, as it was used just once and here. 

I've removed logging, because this class can not create logger for itself. I have no idea why.